src/ERC4973.sol

```diff
@@ -20,12 +21,13 @@ abstract contract ERC4973 is ERC165, IERC721Metadata, IERC4973 {
   constructor(
     string memory name_,
     string memory symbol_
-  ) {
+  ) ERC4973Permit(name_, "1") {
```

Comment:
I presumed that this "1" needs to be hard-coded as the version as it's never subject to change unless it is redeployed. Upon which the version needs to be bumped by the developer. Right?

Reply:
I'd suggest giving the ERC4973 contract creator (the person who calls the constructor) the control to set the variable as they want. So please add to the `ERC4973.contructor` a third argument `string memory version` and pass it to `ERC4973Permit`.